pkg/security/podsecuritypolicy/provider_test.go

```diff
@@ -46,6 +46,8 @@ func TestCreatePodSecurityContextNonmutating(t *testing.T) {
 				Name: "psp-sa",
 			},
 			Spec: extensions.PodSecurityPolicySpec{
+				// TODO this is commented out because it mutates the pod's annotations
+				//SeccompProfiles:          []string{"foo"},
```

Comment:
This comment needs to be read in context of the other method comments.  So this test is ensuring that the provider method is not mutating the pod (which we're breaking with seccomp).  It:

1.  sets up a pod who's SC would be filled in by a PSP during `Generate`
2.  sets up a PSP that ensures every field that can get a default will
3.  runs the pod through `Generate` and compares it against the original

I added the seccomp piece with a `todo` to remind me to take it out when it comes out of alpha but basically the line breaks the assumption that `Generate` is non-mutating.